src/components/backend-ai-console.ts

```diff
@@ -612,6 +612,8 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
    * Open the user preference dialog.
    */
   async _openUserPrefDialog() {
+    let fields = ['email', 'username', 'password', 'full_name'];
+    await globalThis.backendaiclient.user.get(this.user_id, fields);
```

Comment:
@lizable what is the purpose of these two lines?

Reply:
My mistakes. Probably didn't erased in the middle of conflict handling.
I removed it right away.